### PR TITLE
MapAI: Precision in Building Segmentation, Morten Grundetjern

### PR DESCRIPTION
**Pull request by morteg15**
Map AI Image segmentation.

Task 1.

Model is a Unet model that uses resnet50 as an encoder and the weights are from imagenet.
During training it used a learning rate of 1-e4 This gave an increase in performance from the default 1-e3 by 7-8 percent, Using the batch size of 4 also gave a similar increase over using a batch size of 1.
The loss function was Diceloss with an adam optimizer.
It ran for 10 epochs with the best checkpoint being 7. The model where run on cuda gpus

Task 2.

The Model is of type resnet101. It uses a learning rate of 5-e5, batch size of 1, loss function is Diceloss

Note: had some problems running the evaluation pipeline as it was expecting an image size of (500, 500)

### CODE REVIEW

**Comment by Sjyhne** [comment]
Hello! Thank you for your submission. I see that I've forgotten to write in the description that I want you to fill out the contact details in the bottom of the README within your src folder. So if you could just fill it out, and then create a new PR? :smile:
